src/frontend/sharedComponents/DeviceCard.tsx

```diff
@@ -44,30 +54,60 @@ export const DeviceCard = ({
       throw new ExhaustivenessError(deviceConnectionStatus);
   }
 
+  const {navigate} = useNavigationFromRoot();
+  const {formatMessage} = useIntl();
+
   return (
     <TouchableOpacity
       disabled={!onPress || isDisconnected}
       onPress={() => (onPress ? onPress() : {})}
-      style={[styles.container, style]}>
-      <DeviceNameWithIcon
-        name={name}
-        deviceConnectionStatus={deviceConnectionStatus}
-        thisDevice={thisDevice}
-        deviceType={deviceType}
-        deviceId={deviceId}
-        iconSize={75}
-      />
+      style={[style]}>
+      <View style={styles.container}>
+        <DeviceNameWithIcon
+          style={{flexShrink: 1}}
+          name={name}
+          deviceConnectionStatus={deviceConnectionStatus}
+          thisDevice={thisDevice}
+          deviceType={deviceType}
+          deviceId={deviceId}
+          iconSize={75}
+        />
+        <View style={{alignSelf: 'flex-start'}}>
+          {dateAdded && (
+            <Text style={{fontSize: 12, marginBottom: 10}}>
+              <FormattedDate
+                value={new Date(dateAdded)}
+                year="numeric"
+                month="short"
+                day="2-digit"
+              />
+            </Text>
+          )}
+          {thisDevice && (
+            <TouchableOpacity
+              onPress={() => {
+                navigate('HowToLeaveProject');
+              }}>
+              <Text style={{fontSize: 12, color: RED}}>
```

Comment:
based on what i'm seeing with default OS font settings, should:

- increase font weight
- maybe increase font size?
- maybe use `MAGENTA` instead of `RED`?

Currently:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/aa617fad-42b1-49e1-97e9-78c1cdbf51ce">

Design:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/294ab415-f1af-426d-b5e6-15b5345913d1">

Reply:
https://github.com/digidem/comapeo-mobile/pull/674/commits/7704b2a2aa999a780a5aa0c368dc5b197b353e89